storage/cloud-client/public_access_prevention_test.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+import uuid
+
+from google.cloud import storage
+import pytest
+
+import storage_enforce_public_access_prevention
+import storage_get_public_access_prevention
+import storage_unspecify_public_access_prevention
+
+
+@pytest.fixture()
+def bucket():
+    """Yields a bucket that is deleted after the test completes."""
+    # The new projects enforces uniform bucket level access, so
+    # we need to use the old main project for now.
+    original_value = os.environ['GOOGLE_CLOUD_PROJECT']
+    os.environ['GOOGLE_CLOUD_PROJECT'] = os.environ['MAIN_GOOGLE_CLOUD_PROJECT']
+    bucket = None
+    while bucket is None or bucket.exists():
+        bucket_name = "public-access-prevention-{}".format(uuid.uuid4().hex)
+        bucket = storage.Client().bucket(bucket_name)
+    bucket.create()
+    yield bucket
+    time.sleep(3)
+    bucket.delete(force=True)
+    # Set the value back.
+    os.environ['GOOGLE_CLOUD_PROJECT'] = original_value
+
+
+def test_get_public_access_prevention(bucket, capsys):
+    short_name = storage_get_public_access_prevention
```

Comment:
What is the reason for declaring this as a variable?

Reply:
I based these tests off of the [UBLA tests](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/f0a5cbec18011f956c8d5b795837f1ad7b9d7fbf/storage/cloud-client/uniform_bucket_level_access_test.py#L58) and this is how those are written

Reply:
supa long lines, was my rationale for using this lols...

Reply:
Nothing wrong with this, but it's also ok to just type it in long and let the autoformatter deal with it.